Ruqoyah/More-Recipes
# #152748517 Fix Bug in Add Recipe and Mail Notification

#### What does this PR do?
It allows users to be able to see error message when an invalid details is passed to the form and also user will be able to get emails if registered with a valid email

#### Description of Task to be completed?
- ensure redux store is connected appropriately
- ensure a mail notification is sent when user successfully sign up and user recipe was reviewed

#### How should this be manually tested?
- When user signup with a valid email address, he/she should see an email notification
- When the user input an invalid details in the add recipe form, he/she should see an error

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/152748517
